js/apps/account-ui/src/root/Root.tsx

```diff
@@ -37,7 +37,12 @@ const ReferrerLink = () => {
 
 export const Root = () => {
   const { t } = useTranslation();
-  const indexHref = useHref("/");
+  const brandImage = environment.logo ? environment.logo : "logo.svg";
```

Comment:
This can be shortened using a logical disjunction operator.
```suggestion
  const brandImage = environment.logo || "logo.svg";
```